dataland-frontend/src/components/general/CompanyInfoSheet.vue

```diff
@@ -1,8 +1,16 @@
 <template>
   <div ref="sheet" :class="`sheet ${isCollapsed ? 'visuals-hidden' : ''}`" data-test="sheet">
     <template v-if="!useMobileView">
-      <BackButton />
-      <CompaniesOnlySearchBar @select-company="$router.push(`/companies/${$event.companyId}`)" class="w-8 mt-2" />
+      <div :class="` ${showSingleDataRequestButton ? 'headline' : ''}`">
+        <BackButton />
+        <SingleDataRequestButton v-if="showSingleDataRequestButton" :company-id="companyId" />
```

Comment:
This leads to a discrepancy in placement on Company Cockpit compared to Figma

Reply:
addressed